src/btree/bt_io.c

```diff
@@ -365,6 +366,12 @@ __wt_bt_write(WT_SESSION_IMPL *session, WT_ITEM *buf,
 	WT_STAT_FAST_DATA_INCR(session, cache_write);
 	WT_STAT_FAST_CONN_INCRV(session, cache_bytes_write, dsk->mem_size);
 	WT_STAT_FAST_DATA_INCRV(session, cache_bytes_write, dsk->mem_size);
+	if (checkpoint_io) {
```

Comment:
> How does having the absolute bytes written to disk instead of bytes pushed out of cache improve that information?

Often I and other people compare byte written numbers to raw disk numbers. Especially since the iostat numbers are available in the MongoDB diagnostic data.

I hadn't realized that ``bm->write_size()`` exists, that seems like a good solution to me. We'll probably miss counting the root page, but that's OK. I'll add a commit to this ticket.